cmd/identities/import.go

```diff
@@ -1,13 +1,108 @@
 package identities
 
 import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"time"
+
 	"github.com/spf13/cobra"
 
-	"github.com/ory/kratos/cmd/client"
+	"github.com/ory/kratos/internal/clihelpers"
+
+	"github.com/ory/kratos/cmd/cliclient"
+	"github.com/ory/kratos/internal/httpclient/client/admin"
+	"github.com/ory/kratos/internal/httpclient/models"
 )
 
 // importCmd represents the import command
 var importCmd = &cobra.Command{
-	Use: "import <file.json [file-2.json [file-3.json] ...]>",
-	Run: client.NewIdentityClient().Import,
+	Use:   "import <file.json [file-2.json [file-3.json] ...]>",
+	Short: "Import identities from files or STD_IN",
+	Long: `Import identities from files or STD_IN:
```

Comment:
I think the import command should per default be able to import lists of identities. I think it should be possible to detect wether it's a single identity (object) or an array of identities using `gjson`. I assume most people want to import more than one identity.